validator/src/test/java/com/amazon/aoc/models/kubernetes/KubernetesContextFactoryTest.java

```diff
@@ -11,6 +12,13 @@
 
 public class KubernetesContextFactoryTest {
 
+  @Test
+  public void testCreateContextEmptyStrings() throws Exception {
+    KubernetesContextFactory mockFactory = new KubernetesContextFactory("", "", "");
+    KubernetesContext actualKubernetesContext = mockFactory.create();
+    assertNull(actualKubernetesContext);
+  }
```

Comment:
If you have this test here, why are you not instantiating the factory so that `create` returns `null` [here](https://github.com/aws-observability/aws-otel-test-framework/pull/1353/files#diff-d564e22ddf81aaa52b4b0ab7244be15864537545edb3538499eba3f27bebf03dR169).